<div id="horizon_europe" class="d-none">
    <div class="content-page">
        <br />
        <div class="p-2" id="vis-parent" style="margin: auto;">
            <div id="vis" style="width: 35vw; margin: auto;"></div>
        </div>
        <div class="p-2" style="margin: auto;">
            <h1  id="text_start" class="d-flex justify-content-center ">
            
            <div  class="c2 text-center ">Data Management Plan of $_PROJECT  </div>  
            </h1>
            <div class="d-flex justify-content-center">
            
            
        
            <br />
            <table  >
                <tr>
                    <td><h3>Action Number: </h3></td>
                    <td><h3>  $_PROJECT</h3></td>
                </tr>
                <tr>
                    <td><h3>Action Acronym:</h3></td>
                    <td><h3>  $_PROJECT</h3></td>
                </tr>
                <tr>
                    <td><h3>Action Title:</h3></td>
                    <td> <h3>  $_PROJECT</h3></td>
    
                </tr>
                <tr>
                    <td><h3>Date:</h2></td>
                    <td id="today_date" ><h3></h3></td>

                </tr>
                <tr>
                    <td><h3>DMP version:</h3></td>
                    <td id="dmp_version" ><h3> $_DMPVERSION</h3></td>

                </tr>
            </table>

            </div>
            <br />

        </div>
    </div>

    <div class="content-page">
    <h1 class="c13"><span class="c2">1&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"></span><span class="c2">Introduction</span></h1>
    <p class="c0"><span class="c1">#if$_EU The $_PROJECT is part of the Open Data Initiative (ODI) of the EU. #endif$_EU To best profit from open data, it is necessary not only to store data but to make data Findable, Accessible, Interoperable, and Reusable (FAIR).#if$_PROTECT We support open and FAIR data, however, we also consider the need to protect individual data sets. #endif$_PROTECT
        </span>
    </p>
    <p class="c0">
        <span class="c1">
            The aim of this document is to provide guidelines on principles guiding the data management in the $_PROJECT and what data will be stored by using the responses to the EU questionnaire on Data Management Plan (DMP) as a DMP
            document.
        </span>
    </p>
    <p class="c0">
        <span class="c1">
            The detailed DMP instructs how data will be handled during and after the project. The $_PROJECT DMP is modelled according to the Horizon 2020 and Horizon Europe online Manual. #if$_UPDATE It will be updated/its validity checked during the
            $_PROJECT project several times. At the very least, this will happen at month $_UPDATEMONTH. #endif$_UPDATE
        </span>
    </p>
    <h1 class="c13" id="h.wup0l4f7jl9k"><span class="c2">2&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"></span><span class="c2 c12">Data Management Plan EU Template</span></h1>

    <h2 class="c9" id="h.3l7mu272odf4"><span class="c4">2.1&nbsp;&nbsp;&nbsp;&nbsp;Data Summary</span></h2>
    <p class="c0"><span class="c3">Will you re-use any existing data and what will you re-use it for? State the reasons if re-use of any existing data has been considered but discarded.</span></p>
    <p class="c0">

        <span class="c1">
            For the $_PROJECT, data collection#if!$_VVISUALIZATION and integration #endif!$_VVISUALIZATION#if$_VVISUALIZATION, integration and visualization #endif$_VVISUALIZATION #if$_DATAPLANT through the DataPLANT ARC structure are absolutely necessary #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data management process is absolutely necessary #endif!$_DATAPLANT as data is not only used to understand principles, but it is also
            used for analyzing data. In the end, stakeholder involvement needs to be informed about data provenance. It is therefore of importance that not only data is well generated, but also well annotated with metadata
            using open standards, as it is laid out in the following section. As the $_PROJECT aims at $_PROJECTAIM.
            <p class="c0"><span class="c3">What types and formats of data will the project generate/collect?</span></p>
           </span>
          </p>
            <p class="c0">
                <span class="c1">
                    We foresee that the following data will be collected and generated at the very least: $_PHENOTYPIC, $_GENETIC, $_GENOMIC, $_METABOLOMIC, $_RNASEQ, data about $_STUDYOBJECT. In addition, derived data from the
                    original raw data sets will also be collected. This is important, as different analytical pipelines might yield different results or include ad-hoc data analysis parts,#if$_DATAPLANT and these pipelines will be
                    tracked in the DataPLANT ARC#endif$_DATAPLANT. Therefore, specific care will be taken to document and archive these resources (including the analytic pipelines) as well#if$_DATAPLANT relying on the vast expertise
                    in the DataPLANT consortium#endif$_DATAPLANT.
                </span>
            </p>
            <p class="c0"><span class="c3"></span></p>
            <p class="c0"><span class="c3">Will you re-use any existing data and how?</span></p>
            <p class="c0">
                <span class="c1">
                    The project builds on existing data sets and relies on them. #if$_RNASEQ For instance, without a proper genomic reference it is very difficult to analyze NGS data sets.#endif$_RNASEQ It is also important to
                    include existing data sets on the expression and metabolic behaviour of $_STUDYOBJECT, but of course, also on existing characterization and the background knowledge. #if$_PARTNERS of the partners. #endif$_PARTNERS
                    Genomic references can simply be gathered from reference databases for genomes/sequences, like the National Center for Biotechnology Information: NCBI (US); European Bioinformatics Institute: EBI (EU); DNA Data
                    Bank of Japan: DDBJ (JP). Furthermore, prior 'unstructured' data in the form of publications and data contained therein will be used for decision making.
                </span>
            </p>
    </div>
    <div class="content-page">
        <p class="c0"><span class="c3">What is the origin of the data?</span></p>
        <p class="c0"><span class="c1">Public data will be extracted as described in the previous paragraph. For the $_PROJECT, specific data sets will be generated by the consortium partners.</span></p>
        
            
            <p class="c0">
                <span class="c1"> Data of different type or of different domain will be generated differently. For example:</span>
            </p>
            <ul style="list-style-type:disc;">
            #if$_RNASEQ
           <li> <p class="c0">
                <span class="c1"> Short read sequencing will be either collected or outsourced and raw data will be received. </span>
            </p></li>
            #endif$_RNASEQ 
            #if$_METABOLOMIC
            <li><p class="c0">
                <span class="c1"> Metabolomic data will be generated using chromatography coupled to mass spectrometry and from enzyme platforms mostly.</span>
            </p></li>
            #endif$_METABOLOMIC #if$_PROTEOMIC
            <li>
                <p class="c0">
                    <span class="c1"> Proteomic data will be generated using an EU platform which are in line with community standards.</span>
                </p>
            </li>
            #endif$_PROTEOMIC 
            
            #if$_IMAGE
            <li>
                <p class="c0">
                    <span class="c1"> Image data will be generated by using equipment (cameras, scanners, and microscopes) or software. Original images which contain metadata such as exif photo information will be archived.</span>
                </p>
            </li>
            #endif$_IMAGE 
            
            #if$_GENOMIC
            <li>
                <p class="c0">
                    <span class="c1"> Genomic data will be created from sequencing data. The sequencing data will be collected by Next Generation Sequencing (NGS) equipment#if$_PARTNERS or get from parterners#endif$_PARTNERS. Then the sequencing data will be processed to get the genomic data. </span>
                </p>
            </li>
            #endif$_GENOMIC 
            
            #if$_GENETIC
            <li>
                <p class="c0">
                    <span class="c1"> Genetic data will be generated by using Next Generation Sequencing (NGS) equipment. </span>
                </p>
            </li>
            #endif$_GENETIC 
            
            #if$_TARGETED
            <li>
                <p class="c0">
                    <span class="c1"> Targeted assays (e.g. glucose and fructose content) will be generated using specific equipment or experiments. The procedure is fully documented in the lab book. </span>
                </p>
            </li>
            #endif$_TARGETED 
            
            
            #if$_MODELS
            <li>
                <p class="c0">
                    <span class="c1"> The model data will be generated by software simulations. The complete workflow, which includes the environment, runtime, parameters, and results, will be documented and achieved. </span>
                </p>
            </li>
            #endif$_MODELS
            
            #if$_CODE
            <li>
                <p class="c0">
                    <span class="c1"> The code data will be generated by programmers. </span>
                </p>
            </li>
            #endif$_CODE
            
            #if$_EXCEL
            <li>
                <p class="c0">
                    <span class="c1"> The Excel data will be generated by experimentalist or data analyst by using Office or opensource software. </span>
                </p>
            </li>
            #endif$_EXCEL
            
            #if$_CLONED-DNA
            <li>
                <p class="c0">
                    <span class="c1"> The cloned DNA data will be generated by using a sequencing tool. </span>
                </p>
            </li>
            #endif$_CLONED-DNA
            
            #if$_PHENOTYPIC
            <li>
                <p class="c0">
                    <span class="c1"> Phenotypic data will be generated using phenotyping platforms. </span>
                </p>
            </li>
            #endif$_PHENOTYPIC
            </ul>

        #if$_PREVIOUSPROJECTS
        <p class="c0"><span class="c1"> Data from previous projects such as $_PREVIOUSPROJECTS will be considered. </span></p>
        #endif$_PREVIOUSPROJECTS
        <p class="c0"><span class="c3">What is the expected size of the data?</span></p>
        <p class="c0"><span class="c1">We expect to generate raw data in the range of $_RAWDATA GB of data. The size of the derived data will be about $_DERIVEDDATA GB.</span></p>
        <p class="c0"><span class="c3"></span></p>
        <p class="c0"><span class="c3">To whom might it be useful ('data utility')?</span></p>
        <p class="c0">
            <span class="c1">
                The data will be useful for the $_PROJECT partners, the scientific community working on $_STUDYOBJECT or the general public interested in $_STUDYOBJECT. Hence, the $_PROJECT also strives to collect the data
                that has been disseminated and potentially advertise it#if$_DATAPLANT e.g. through the DataPLANT platform or other means#endif$_DATAPLANT, if it is not included in a publication anyway, which is the most
                likely form of dissemination.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                $_DATAUTILITY
            </span>
        </p>
        <h2 class="c9" id="h.8sk9nau3ts35"><span class="c4">2.2&nbsp;&nbsp;&nbsp;&nbsp;FAIR data</span></h2>
        <h3 class="c14" id="h.gtd4kdf6hncf"><span class="c7">Making data findable, including provisions for metadata</span></h3>
        <p class="c0">
            <span class="c3">
                Are the data produced and/or used in the project discoverable with metadata, identifiable and locatable by means of a standard identification mechanism (e.g. persistent and unique identifiers such as Digital
                Object Identifiers)?
            </span>
        </p>
        <p class="c0">
                <span class="c1">
                    All data sets will receive unique identifiers, and they will be annotated with metadata.#if$_MIAPPE The $_PROJECT will rely on community standards plus additional recommendations necessary in plant science adapted by e.g. using suggestions from the Minimum Information About a Plant Phenotyping Experiment (MIAPPE). #endif$_MIAPPE </span>
                    <span class="c1">These unlike cross-domain minimal sets such as Dublin core, which mostly defines the submitter and what general type of data is being dealt with (e.g. images), allow reusability by other researchers as it also defines properties of the plant (see the preceding section). However, of course minimal cross-domain annotations are part of the $_PROJECT. #if$_DATAPLANT The core integration with DataPLANT will also allow one to tag individual releases with a Digital Object Identifier (DOI).
                    #endif$_DATAPLANT #if$_OTHERSTANDARDS $_OTHERSTANDARDINPUT #endif$_OTHERSTANDARDS
                </span>
            </p>
        <p class="c0"><span class="c3">What naming conventions do you follow?</span></p>
        <p class="c0">
            <span class="c1">
                Data variables will use standard names. For example, this is the case for genes, metabolites, and proteins. These will also be linked to free biomedical ontologies where possible. In the case of
                datasets, the dataset names will also be made to be meaningful and human readable. In addition, traditional names will of course be included where necessary as synonyms.
            </span>
        </p>
        <p class="c0"><span class="c3">Will search keywords be provided that optimize possibilities for re-use?</span></p>
        <p class="c0">
            <span class="c1">
                Keywords about the experiment and the general consortium will be included, as well as an abstract about the data, where useful. In addition, certain keywords can be auto-generated from dense metadata and its
                underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with standardized DataPLANT ontologies that are supplemented where the ontology does not yet include the variables.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0"><span class="c3">Do you provide clear version numbers?</span></p>
        <p class="c0">
            <span class="c1">
                To maintain data integrity and to be able to re-analyze data, data sets will get version numbers where this is useful (e.g. raw data must not be changed and will not get a version number and is considered
                immutable). #if$_DATAPLANT This is automatically supported by the ARC Git DataPLANT infrastructure. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0">
            <span class="c3">What metadata will be created? In case metadata standards do not exist in your discipline, please outline what type of metadata will be created and how.</span><span class="c12 c17 c11"></span>
        </p>
        <p class="c0">
            <span class="c1">We foresee to use Investigation, Study, Assay (ISA) specification for metadata creation. #if$_RNASEQ|$_GENOMIC For specific data, e.g., RNAseq or genomic data, we use metadata template from the end-point repositories. </span><span class="c16">#if$_MINSEQE The Minimum Information About a Next-generation Sequencing Experiment (MinSEQe) will also be used. </span>
            <span class="c1">
             #endif$_MINSEQE #endif$_RNASEQ|$_GENOMIC #if$_METABOLOMIC The Metabolights submission compliant standards are used for metabolomic data.#issuewarning some Metabolomics partners considers Metabolights
                not an accepted standard#endissuewarning#endif$_METABOLOMIC The plant community use #if$_MIAPPE MIAPPE  for phenotyping data in the broadest sense, but we will also be relying on #endif$_MIAPPE specific SOPs for additional
                annotations #if$_DATAPLANT relying on advanced DataPLANT annotation and ontologies. #endif$_DATAPLANT
            </span>
            </p>
    </div>
    <div class="content-page">
        <h3 class="c14" id="h.8wrxcmaxl0hv"><span class="c7">Making data openly accessible</span></h3>
        <p class="c0">
            <span class="c3">
                Which data produced and/or used in the project will be made openly available as the default? If certain datasets cannot be shared (or need to be shared under restrictions), we explain why, clearly separating
                legal and contractual reasons from voluntary restrictions.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                By default, all data sets from the $_PROJECT will be shared with the community and made openly available. This is, however, after partners have had the ability to check for IP protection (according to agreements
                and background rights). #if$_INDUSTRY This applies in particular to data pertaining to the industry. #endif$_INDUSTRY However, all partners also strive for IP protection of data sets which will be tested and
                due diligence will be given.
            </span>
        </p>
        <p class="c0">
            <span class="c3">
                Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their data closed if relevant provisions are made in the consortium agreement and are in line with the reasons
                for opting out.
            </span>
        </p>
        <p class="c0"><span class="c3"></span></p>
        <p class="c0"><span class="c3">How will the data be made accessible (e.g. by deposition in a repository)?</span></p>
        <p class="c0">
             <p class="c0">
                <span class="c1">
                    #if!$_DATAPLANT Data will be made available via the $_PROJECT platform using a user-friendly front end that allows data visualization. Besides this it will be ensured that data which can be stored in
                    international discipline related repositories which use specialized technologies (Sequencing at the national US center $_NCBI, $_GEO; Sequencing and sequence data at the EU center: EBI, $_ENA; Proteome database:
                    $_PRIDE; metabolomic database: $_METABOLIGHTS; #if$_OTHEREP and $_OTHEREP #endif$_OTHEREP )  will be used to store data and the data will be processed there as well. #endif!$_DATAPLANT
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    As noted above, specialized repositories like SRA /ENA, Pride /Proteomexchange are the most common ones and will be used when appropriate. In the case of unstructured less standardized data (e.g. experimental
                    phenotypic measurements), these will be metadata annotated and if complete given a digital object identifier (DOI). #if$_DATAPLANT and the whole data sets wrapped into an ARC will get DOIs as well. The ARC and the converters provided by DataPLANT will guarantee the upload into the endpoint repositories is fast and easy.
                    #endif$_DATAPLANT 
                </span>
            </p>
        </p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c3">What methods or software tools are needed to access the data?</span></p>
        <p class="c0"><span class="c1">#if$_PROPRIETARY The $_PROJECT relies on the tool(s) $_PROPRIETARY. #endif$_PROPRIETARY</span></p>
        <p class="c0">
            <span class="c1">
                #if!$_PROPRIETARY No specialized software will be needed to access the data, usually just a modern browser. Access will be possible through web interfaces. For data processing after obtaining raw data, typical
                open-source software can be used. #endif!$_PROPRIETARY
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT DataPLANT offers tools such as the open-source SWATE plugin for Excel, the ARC commander, and the DMP tool which will not necessarily make the interaction with data more convenient.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0"><span class="c3">Is documentation about the software needed to access the data included?</span></p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT However, DataPLANT resources are well described, and their setup is documented on their github project pages. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0"><span class="c3">Is it possible to include the relevant software (e.g. in open-source code)?</span></p>
        <p class="c0"><span class="c1">As stated above, here we use publicly available open-source and well-documented certified software #if$_PROPRIETARY except for $_PROPRIETARY #endif$_PROPRIETARY.</span></p>
        <p class="c0"><span class="c12 c11 c17"></span></p>
        <p class="c0">
            <span class="c3">Where will the data and associated metadata, documentation and code be deposited? Preference should be given to certified repositories that support open access, where possible.</span>
        </p>
        <p class="c0">
            <span class="c1">
                As noted above, specialized repositories like SRA /ENA, Pride /Proteomexchange are the most common ones and will be used when appropriate. In the case of unstructured less standardized data (e.g. experimental
                phenotypic measurements), these will be metadata annotated and if complete given a digital object identifier (DOI) #if$_DATAPLANT and the whole data sets wrapped into an ARC will get DOIs as
                well#endif$_DATAPLANT.
            </span>
        </p>
        <p class="c0"><span class="c3">Have you explored appropriate arrangements with the identified repository?</span></p>
        <p class="c0">
            <span class="c1">
                The submission is for free, and it is the goal (at least of ENA) to obtain as much data as possible. Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required.
                #if$_DATAPLANT For DataPLANT, this has been agreed upon. #issuewarning Please use endpoint repositories to store or archive your data after publication. #endissuewarning #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0"><span class="c3">If there are restrictions on use, how will access be provided?</span></p>
        <p class="c0"><span class="c1">There are no restrictions, beyond the aforementioned IP checks, which are in line with e.g. European open data policies.</span></p>
        <p class="c0"><span class="c7"></span></p>
    </div>
    <div class="content-page">
        <p class="c0"><span class="c3">Is there a need for a data access committee?</span></p>
        <p class="c0"><span class="c1">Consequently, there is no need for a committee.</span></p>
        <p class="c0"><span class="c3">Are there well described conditions for access (i.e. a machine-readable license)?</span></p>
        <p class="c0"><span class="c1">Yes, where possible, e.g. CC REL will be used for data not submitted to specialized repositories such as ENA.</span></p>
        <p class="c0"><span class="c3">How will the identity of the person accessing the data be ascertained?</span></p>
            <p class="c0">
                <span class="c1">
                    In case data is only shared within the consortium, if the data is not yet finished or under IP checks, the data is hosted internally and username and password will be required (see also our GDPR rules). In
                    the case data is made public under final EU or US repositories, completely anonymous access is normally allowed. This is the case for ENA as well and both are in line with GDPR requirements.
                </span>
            </p>
                    <p class="c0">
<span class="c1">
    #if$_DATAPLANT Currently, data management relies on the annotated research context ARC. It is password protected, so before any data can be obtained or samples, generated an authentication needs to take place.
    #endif$_DATAPLANT
</span>
</p>
        <h3 class="c14" id="h.514g6glbmw6i"><span class="c7">Making data interoperable</span></h3>
        <p class="c10">
            <span class="c3">
                Are the data produced in the project interoperable, that is allowing data exchange and re-use between researchers, institutions, organizations, countries, etc. (i.e. adhering to standards for formats, as much
                as possible compliant with available (open) software applications, and in particular facilitating re-combinations with different datasets from different origins)?
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                Whenever possible, data will be stored in common and openly defined formats including all the necessary metadata to interpret and analyze data in a biological context. By default, no proprietary formats will be
                used; however Microsoft Excel files (according to ISO/IEC 29500-1:2016) might be used as intermediates by the consortium#if$_DATAPLANT and by some ARC components in form#endif$_DATAPLANT. In addition, text
                files might be edited in text processor files, but will be shared as pdf.
            </span>
        </p>
        <p class="c10"><span class="c3">What data and metadata vocabularies, standards or methodologies will you follow to make your data interoperable?</span></p>
        <p class="c0">
            <span class="c1">
                As mentioned above, we foresee using #if$_RNASEQ|$_GENOMIC e.g. #if$_MINSEQE MinSEQe for sequencing data and #endif$_MINSEQE #endif$_RNASEQ|$_GENOMIC Metabolights compatible forms for metabolites #if$_MIAPPE  
                as well as MIAPPE for phenotyping-like data #endif$_MIAPPE. The latter will thus allow the integration of data across projects and safeguards that reuse established and tested protocols. Additionally, we will use
                ontology terms to enrich the data sets relying on free and open ontologies. In addition, additional ontology terms might be created and be canonized during the $_PROJECT.
            </span>
        </p>
        <p class="c10"><span class="c3">Will you be using standard vocabularies for all data types present in your data set, to allow inter-disciplinary interoperability?</span></p>
        <p class="c0">
            <span class="c1">
                In fact, open biomedical ontologies will be used where they are mature. As stated in the previous question, sometimes ontologies and controlled vocabularies might have to be extended. #if$_DATAPLANT Here, the
                $_PROJECT will build on the advanced ontologies developed in DataPLANT. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c10"><span class="c3">In case it is unavoidable that you use uncommon or generate project specific ontologies or vocabularies, will you provide mappings to more commonly used ontologies?</span></p>
        <p class="c0"><span class="c1">Common and open ontologies will be used, hence this question does not apply.</span></p>
        <h3 class="c14" id="h.fs2sjyqtlku7"><span class="c7">Increase data reuse (by clarifying licences)</span></h3>
        <p class="c10"><span class="c12 c17 c11"></span></p>
        <p class="c10"><span class="c3">How will the data be licensed to permit the widest re-use possible?</span></p>
        <p class="c0"><span class="c1">Open licenses, such as Creative Commons CC, will be used whenever possible.</span></p>
    </div>
    <div class="content-page">
        <p class="c10">
            <span class="c3">
                When will the data be made available for re-use? If an embargo is sought to give time to publish or seek patents, specify why and how long this will apply, bearing in mind that research data should be made
                available as soon as possible.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_early The data will be published as soon as possible to guarantee reusability. #endif$_early #if$_ipissue In general, IP issues will first be checked. #endif$_ipissue All consortium partners will be
                encouraged to make data available prior to publication openly and/or under pre-publication agreements #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto International Data
                Release Workshop. #endif$_GENOMIC
            </span>
        </p>
        <p class="c10"><span class="c3">Are the data produced and/or used in the project usable by third parties, in particular after the end of the project? If the re-use of some data is restricted, explain why.</span></p>
        <p class="c0"><span class="c1">There will be no restrictions once the data is made public.</span></p>
        <p class="c10"><span class="c3">How long is it intended that the data remains re-usable?</span></p>
        <p class="c0"><span class="c1">Data will be made available for many years#if$_DATAPLANT and potentially indefinitely after the end of the project#endif$_DATAPLANT.</span></p>
        <p class="c0">
            <span class="c1">In any case data submitted to  repositories (as detailed above) e.g. ENA /PRIDE would be subject to local data storage regulation.</span>
        </p>
        <p class="c10"><span class="c3">Are data quality assurance processes described?</span></p>
        <p class="c0">
            <span class="c1">
                The data will be checked and curated. #if$_DATAPLANT Furthermore, data will be analyzed for quality control (QC) problems using automatic procedures as well as by manual curation #endif$_DATAPLANT.
            </span>
        </p>
        <h2 class="c9" id="h.w5pimtt9qf6p"><span class="c4">2.3&nbsp;&nbsp;&nbsp;&nbsp;Allocation of resources</span></h2>
        <p class="c10"><span class="c3">What are the costs for making data FAIR in your project?</span></p>
        <p class="c0"><span class="c1">The costs comprise data curation, #if$_DATAPLANT ARC consistency checks, #endif$_DATAPLANT and maintenance on the $_PROJECT´s side.</span></p>
        <p class="c0">
            <span class="c1">
                Additionally, last-level costs for storage are incurred by last-level repositories (e.g. ENA) but not charged against the $_PROJECT or its members but by the operation budget of these repositories.
            </span>
        </p>
        <p class="c10">
            <span class="c3">How will these be covered? Note that costs related to open access to research data are eligible as part of the Horizon 2020 or Horizon Europe grant (if compliant with the Grant Agreement conditions).</span>
        </p>
        <p class="c0"><span class="c1">A large part of the cost is covered by the $_PROJECT. #if$_DATAPLANT The structures tools, and knowledge laid down in the DataPLANT consortium. #endif$_DATAPLANT</span></p>
        <p class="c10"><span class="c3">Who will be responsible for data management in your project?</span></p>
        <p class="c0"><span class="c1">The responsible will be $_DATAOFFICER.</span></p>
        <p class="c10"><span class="c3">Are the resources for long term preservation discussed (costs and potential value, who decides and how/what data will be kept and for how long)?</span></p>
        <p class="c0">
            <span class="c1">
                The data responsible(s) (data officer #if$_PARTNERS or $_PARTNERS #endif$_PARTNERS) decides on preservation of data not submitted to end-point subject area repositories #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT after
                project end. This will be in line with EU, institute policies, and data sharing based on EU and international standards.
            </span>
        </p>
        <h2 class="c9" id="h.of351jlpnl2l"><span class="c4">2.4&nbsp;&nbsp;&nbsp;&nbsp;Data security</span></h2>
        <p class="c10"><span class="c3">What provisions are in place for data security (including data recovery as well as secure storage and transfer of sensitive data)?</span></p>
        <p class="c20">
            <span class="c1">
                Once data is transferred to the $_PROJECT platform#if$_DATAPLANT and ARCs have been generated in DataPLANT#endif$_DATAPLANT, data security will be imposed. This comprises secure storage, and the use of
                passwords and usernames is generally transferred via separate safe media.
            </span>
        </p>
        <p class="c10"><span class="c3">Is the data safely stored in certified repositories for long term preservation and curation?</span></p>
        <p class="c0">
            <span class="c1">
                Wherever there are certified repositories, these will be used as end-point repositories. #if$_RNASEQ Transcriptomics data and gene sequence data will be also made available upon publication via the standards
                ENA/SRA, #endif$_RNASEQ #if$_METABOLOMIC metabolite data in e.g. Metabolights (and/or Nationwide repositories like the German NFDI the French INRAe), #endif$_METABOLOMIC #if$_PROTEOMIC Proteomics data in
                e.g. Pride/Proteomexchange #endif$_PROTEOMIC. In addition, the national resource will maintain safekeeping of data also after the project ends. #if$_DATAPLANT In addition, databases like e.g. Proteomexchange
                do not support deep plant specific metadata; hence ARCs will be maintained to ensure reusability. #endif$_DATAPLANT
            </span>
        </p>
        <h2 class="c9" id="h.ozaz4qpzixaa"><span class="c4">2.5&nbsp;&nbsp;&nbsp;&nbsp;Ethical aspects</span></h2>
        <p class="c10">
            <span class="c3">
                Are there any ethical or legal issues that can have an impact on data sharing? These can also be discussed in the context of an ethics review. If relevant, include references to ethics deliverables and
                ethics chapter in the Description of the Action (DoA).
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                At the moment, we do not anticipate ethical or legal issues with data sharing. In terms of ethics, since this is plant data, there is no need for an ethics committee, however, diligence for plant resource benefit
                sharing is considered (🡺see Nagoya protocol). #issuewarning You have to check here and enter any due diligence here at the moment we are awaiting if Nagoya gets also part of sequence information. In any case
                if you use material not from your (partner) country and characterize this physically e.g., metabolites, proteome, biochemically RNASeq etc. this might represent a Nagoya relevant action unless this is from e.g.
                US (non partner), Ireland (not signed still contact them) etc but other laws might apply…. #endissuewarning
            </span>
        </p>
        <p class="c10"><span class="c3">Is informed consent for data sharing and long term preservation included in questionnaires dealing with personal data?</span></p>
        <p class="c0">
            <span class="c1">
                The only personal data that will potentially be stored is the submitter name and affiliation in the metadata for data. In addition, personal data will be collected for dissemination and communication
                activities using specific methods and procedures developed by the $_PROJECT partners to adhere to data protection. #issuewarning You need to inform and better get WRITTEN consent that you store emails and
                names or even pseudonyms such as twitter handles, we are very sorry about these issues we didn’t invent them #endissuewarning
            </span>
        </p>
        <h2 class="c9" id="h.slb0g3rbvjfl"><span class="c4">2.6&nbsp;&nbsp;&nbsp;&nbsp;Other issues</span></h2>
        <p class="c0"><span class="c3">Do you make use of other national/funder/sectorial/departmental procedures for data management? If yes, which ones?</span></p>
        <p class="c0"><span class="c1">Yes, the $_PROJECT will use common Research Data Management (RDM) tools#if$_DATAPLANT and in particular resources developed by the NFDI of Germany#endif$_DATAPLANT.</span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c6"><span class="c1"></span></p>
        <h1 class="c13" id="h.f589zyp9ll5e"><span class="c2">3&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"> </span><span class="c12 c2">Annexes</span></h1>
        <p class="c0"><span class="c1"></span></p>
        <h2 class="c9" id="h.vrmh31avvatp"><span class="c11 c18">3.1&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"> </span><span class="c4">Abbreviations</span></h2>
        <p class="c0"><span class="c1"></span></p>
        #if$_DATAPLANT<p class="c0"><span class="c11">ARC</span><span class="c1"> Annotated Research Context</span></p> #endif$_DATAPLANT
        
        <p class="c0"><span class="c11">CC</span><span class="c1"> Creative Commons</span></p>
        <p class="c0"><span class="c11">CC CEL</span><span class="c1"> Creative Commons Rights Expression Language</span></p>
        <p class="c0"><span class="c11">DDBJ</span><span class="c1"> DNA Data Bank of Japan</span></p>
        <p class="c0"><span class="c11">DMP</span><span class="c1"> Data Management Plan</span></p>
        <p class="c0"><span class="c11">DoA</span><span class="c1"> Description of Action</span></p>
        <p class="c0"><span class="c11">DOI</span><span class="c1"> Digital Object Identifier</span></p>
        <p class="c0"><span class="c11">EBI</span><span class="c1"> European Bioinformatics Institute</span></p>
        <p class="c0"><span class="c11">ENA</span><span class="c1"> European Nucleotide Archive</span></p>
        <p class="c0"><span class="c11">EU </span><span class="c1">European Union</span></p>
        <p class="c0"><span class="c11">FAIR </span><span class="c1">Findable Accessible Interoperable Reproducible</span></p>
        <p class="c0"><span class="c11">GDPR</span><span class="c1"> General data protection regulation (of the EU)</span></p>
        <p class="c0"><span class="c11">IP</span><span class="c1"> Intellectual Property</span></p>
        <p class="c0"><span class="c11">ISO</span><span class="c1"> International Organization for Standardization</span></p>
        <p class="c0"><span class="c11">MIAMET</span><span class="c1"> Minimal Information about Metabolite experiment</span></p>
        <p class="c0"><span class="c11">MIAPPE</span><span class="c1"> Minimal Information about Plant Phenotyping Experiment</span></p>
        <p class="c0"><span class="c11">MinSEQe</span><span class="c1"> Minimum Information about a high-throughput Sequencing Experiment</span></p>
        <p class="c0"><span class="c11">NCBI</span><span class="c1"> National Center for Biotechnology Information</span></p>
        <p class="c0"><span class="c11">NFDI</span><span class="c1"> National Research Data Infrastructure (of Germany)</span></p>
        <p class="c0"><span class="c11">NGS</span><span class="c1"> Next Generation Sequencing</span></p>
        <p class="c0"><span class="c11">RDM</span><span class="c1"> Research Data Management</span></p>
        <p class="c0"><span class="c11">RNASeq</span><span class="c1"> RNA Sequencing</span></p>
        <p class="c0"><span class="c11">SOP</span><span class="c1"> Standard Operating Procedures</span></p>
        <p class="c0"><span class="c11">SRA</span><span class="c1"> Short Read Archive</span></p>
    #if$_DATAPLANT<p class="c0"><span class="c11">SWATE</span><span class="c1"> Swate Workflow Annotation Tool for Excel</span></p> #endif$_DATAPLANT
        <p class="c0"><span class="c11">ONP</span><span class="c1"> Oxford Nanopore</span></p>
        <p class="c0"><span class="c11">qRT</span><span class="c11">PCR</span><span class="c1"> quantitative real time polymerase chain reaction</span></p>
        <p class="c0"><span class="c11">WP</span><span class="c1"> Work Package</span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c22"><span class="c1"></span></p>
</div>
</div>